pkg/ucp/store/object.go

```diff
@@ -23,10 +23,8 @@ import (
 type ETag = string
 
 type Metadata struct {
-	ID          string
-	ETag        ETag
-	APIVersion  string
-	ContentType string
+	ID   string
+	ETag ETag
```

Comment:
Can you pls give more info on this change?

Reply:
Sure. The `APIVersion` and `ContentType` fields have been part of the database implementation for a long time. `APIVersion` is redundant with a field in the datamodel. `ContentType` has always been unused. 

You'll notice that I only had to update test code to remove this - and one of the DB implementations. Nothing reads these fields.